tests/integration/views/surveys/test_upload_survey.py

```diff
@@ -110,3 +122,9 @@ def test_upload_survey_content_too_long(self):
 
         self.assertEqual(response.status_code, 302)
         self.assertTrue('/surveys/upload-failed'.encode() in response.data)
+
+    def test_is_collection_instrument_too_small(self):
```

Comment:
Reading this test, I realise the logic is the wrong way around.  I would expect a functionc alled `is_collection_instrument_too_small` to return True if it IS too small, and False if it IS NOT too small.  

The way it's done currently is the opposite of that.